solidity/test/BasicConsumer_test.js

```diff
@@ -46,7 +47,7 @@ contract('BasicConsumer', () => {
 
     context('with LINK', () => {
       beforeEach(async () => {
-        await link.transfer(cc.address, web3.toWei('1', 'ether'))
+        await link.transfer(cc.address, web3.utils.toWei('1', 'ether'))
```

Comment:
We have a `toWei` helper method that we should probably use here, instead of switching all of these every time the truffle API changes. Then, we can just switch the implementation of the toWei helper.

Reply:
974994f3